scalafix-tests/input/src/main/scala-2/test/explicitResultTypes/ExplicitResultTypesSingleton.scala

```diff
@@ -1,5 +1,6 @@
 /*
 rules = ExplicitResultTypes
+ExplicitResultTypes.skipSimpleDefinitions = []
```

Comment:
the singletons in this tests are a `Term.Ref`, hence for this unit test to work the config needs to be overwritten